common/views/components/Quote/Quote.js

```diff
@@ -1,35 +1,22 @@
 // @flow
-import {Fragment} from 'react';
 import {font} from '../../../utils/classnames';
 import PrismicHtmlBlock from '../PrismicHtmlBlock/PrismicHtmlBlock';
 import type {HTMLString} from '../../../services/prismic/types';
 
 type Props = {|
-  body: HTMLString,
-  footer: ?string,
-  citationLink: ?string
+  text: HTMLString,
+  citation: ?HTMLString
 |}
 
-const Quote = ({body, footer, citationLink}: Props) => (
+const Quote = ({text, citation}: Props) => (
   <blockquote
     className={`quote quote--block ${font({s: 'HNL3'})}`}>
-    {/*
-      TODO (wordpress migration); This is because we don't get structured
-      content from WP.
-    */}
-    {typeof body === 'string'
-      ? <div dangerouslySetInnerHTML={{__html: body}} />
-      : <PrismicHtmlBlock html={body} />
-    }
+    <PrismicHtmlBlock html={text} />
 
-    {footer &&
-      <footer className='quote__footer'>
-        <cite className={`quote__cite ${font({s: 'HNL5'})}`}>
-          {citationLink
-            ? <a href={citationLink}>
-              {footer}
-            </a> : <Fragment>{footer}</Fragment>
-          }
+    {citation &&
+      <footer className='quote__footer flex'>
+        <cite className={`quote__cite flex ${font({s: 'HNL5'})}`} style={{alignItems: 'flex-end'}}>
```

Comment:
I think I'd be happier with a `flex--v-end` class added to the root-scope-classes